cpp/open3d/pipelines/registration/Feature.cpp

```diff
@@ -32,7 +32,7 @@ static Eigen::Vector4d ComputePairFeatures(const Eigen::Vector3d &p1,
     auto n2_copy = n2;
     double angle1 = n1_copy.dot(dp2p1) / result(3);
     double angle2 = n2_copy.dot(dp2p1) / result(3);
-    if (acos(fabs(angle1)) > acos(fabs(angle2))) {
+    if (fabs(angle1) < fabs(angle2)) {
```

Comment:
@theNded Could you explain why you said this has no performance impact and its a "typo fix"?

Reply:
This is definetely a sub-millisecond operation, so there would not be a performance impact at all.
It is a "typo fix" in the math sense. Nothing is really wrong, it just becomes slightly neater.

Of course, I encourage such minor fixes! But as said, it might be better to accumulate multiple such changes to make a PR altogether. This will help maintain a neat git history and stable CI usage.

Reply:
Thanks. Makes sense.